core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -184,21 +189,23 @@ trait EntityComponent {
     }
 
     def entityAttributes(entityId: Rep[Long]): EntityAttributeQuery = for {
-      entityAttrRec <- entityAttributeQuery if entityAttrRec.ownerId === entityId
+      entityAttrRec <- entityAttributeQuery if entityAttrRec.ownerId === entityId && ! entityAttrRec.deleted
```

Comment:
is this your bug? this is either hard to read or it's a bit funny. shouldn't there be a filter in here? `entityAttributeQuery.filter( _.entity_id == entityId )` or something? where is `entityAttrRec` coming from on the RHS here?